src/Controllers/App/SitemapController.php

```diff
@@ -201,7 +210,21 @@ function ($content) use ($type) {
         return new WP_REST_Response($data);
     }
 
-    private function getTotalPostCount($postType) {
+    /**
+     * Get an array of post tags term ids,
+     * that have less than five posts attached to it.
+     *
+     * @return array
+     */
+    private function getExcludedPostTagsIds()
+    {
+        return collect(get_terms('post_tag'))->reject(function (\WP_Term $tag) {
```

Comment:
hmm perhaps just ask for ids and count if you are not using the other fields, to make the query faster.
```suggestion
        return collect(get_terms([
            'taxonomy' => 'post_tag',
            'fields' => [
                'term_id',
                'count',
            ]
        ]))->reject(function (\WP_Term $tag) {
```